### ISSUE TITLE
Support for non-NodeJS workflows

### ISSUE THREAD

**Issue by harmw** [magneticio/vamp-workflow-agent#7]
At some point I can imagine people wanting workflows written in Python, instead of being forced to using NodeJS 👍 This will of course require a `vamp-python-client`...

**Comment by olafmol** [comment]
You can deploy any container as a vamp workflow. The only thing the vamp nodejs api client adds are some higher level methods to agregate and calculate through the vamp api structures. This can easily be done (or ported) to any other scripting/programming language. Of course when there is a lot of demand for a specific language we can provide a specific API library.

**Comment by harmw** [comment]
Yep, I can spin my own workflow container - totally true, but I'd like for stuff to be smarter. Since currently the agent will download a breed (from Consul) and run it through the node binary. It would be nice if running python code would become just as easy and straight-forward. But yes, it's probably not _the_ most important feature to add :)

**Comment by dragoslav** [comment]
Support for other (scripting) languages requires:

- call different binary (https://github.com/magneticio/vamp-workflow-agent/blob/master/run.go#L52)
- allow other workflow breed types (Vamp does validation, so something like `application/python` will be discarded as an error)
- new Docker images or update of the existing with Python etc.

Any volunteer for `vamp-workflow-agent` and `vamp-python-client`? I can update Vamp itself (additional workflow breed types).
